shared_model/interfaces/iroha_internal/transaction_sequence_factory.cpp

```diff
@@ -16,8 +16,17 @@
 
 namespace shared_model {
   namespace interface {
+    namespace {
+      // we use an unnamed namespace here because we don't want to add test as
+      // include path for the shared_model_interfaces_factories target
+      const uint64_t kTestsMaxBatchSize(10000);
```

Comment:
As you said, this hardco~~r~~de is planned to be removed. Please put a TODO for this task.